packages/container-registry-v5/lib/sanbashi.js

```diff
@@ -151,6 +151,12 @@ Sanbashi.cmd = function (cmd, args, options = {}) {
         stdout += data.toString()
       })
     }
+    child.on('error', (err) => {
+      if (err.code === 'ENOENT' && err.path === 'docker') {
```

Comment:
This isn't handling other errors now, is it?

Reply:
@jdxcode I believe it will only handle `ENOENT` errors where it's failing because it can't find `docker` in the path.  But now I'm wondering if having `child.on('error', (err) => {` will disrupt the flow down below where it checks the exit code.  I'm inclined to think that both will still be executed, but I'll double check.

Reply:
I'm pretty sure this will swallow other errors

Reply:
@jdxcode I going to proceed with merging this change, then file a separate PR that I'm actively working on for https://github.com/heroku/cli/issues/1082.  Sound good?